ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/SvarUtConfiguration.java

```diff
@@ -0,0 +1,37 @@
+package no.difi.meldingsutveksling.ks;
+
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.ks.mapping.HandlerFactory;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.oxm.jaxb.Jaxb2Marshaller;
+
+@Configuration
+@EnableConfigurationProperties({IntegrasjonspunktProperties.class})
+public class SvarUtConfiguration {
+    private IntegrasjonspunktProperties properties;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "properties" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)